### DISCUSSION TITLE
How convert config in HTML

### DISCUSSION THREAD

**Discussion by darfilp** [facebook/lexical#3877] [Q&A]
I receive this config from the server. Can I convert it to html? 

{"root":{"children":[{"children":[{"detail":0,"format":1,"mode":"normal","style":"","text":"kjkjkkjkjkjkjkjkjkj","type":"text","version":1}],"direction":"ltr","format":"","indent":0,"type":"paragraph","version":1}],"direction":"ltr","format":"","indent":0,"type":"root","version":1}}

And display it in another component

**Comment by acywatson** [comment]
Yea you can either parse the JSON and traverse it to build up HTML or you can pass that state into a Lexical editor instance and then use Lexical's HTML serializer:

https://lexical.dev/docs/concepts/serialization#lexical---html
